client/src/panels/result_graphs/indicator_display.js

```diff
@@ -0,0 +1,64 @@
+
+import { ensure_loaded } from '../../core/lazy_loader.js';
+import { StandardRouteContainer } from '../../core/NavComponents.js';
+
+import { TM, text_maker } from './result_text_provider.js';
+
+import {
+  Panel,
+  SpinnerWrapper,
+} from '../../components';
+
+import { Indicator } from '../../models/results.js';
+import { SingleIndicatorDisplay } from './result_components.js';
+
+export default class IndicatorPanel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { loading: true };
+  }
+
+  componentDidMount(){
+    const {
+      match: {
+        params: { id },
+      },
+    } = this.props;
+    ensure_loaded({
+      subject: {id: id, level: "indicator"},
+      results: true,
+    })
```

Comment:
Might be best to break away from the old `ensure_loaded` paradigm and have this panel just directly make its own GraphQL request.